.github/workflows/feature_6_docs.yml

```diff
@@ -0,0 +1,63 @@
+name: Develop > Tiny docs 6.0
+
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - '.github/workflows/**'
+      - '*'
+    branches:
+      - 'feature-6/**'
```

Comment:
This goes against the branching strategy being used here and elsewhere... so I don't think adding `6` like this is a good idea. I'm not sure what's better, but I'd prefer either `feature/6/<JIRA>`, `feature/<JIRA>_6.x` or even better just make it work for any `feature/<JIRA>` branch. It might need a wider discussion to be honest.

Reply:
currently feature/ is mapped to the legacy 5x docs
feature-6/ new 6.0
feature-5/ new 5.0

Reply:
Yeah, as I said that's only going to cause issues going forward. We need to keep it `feature/` as the prefix always. If the sole reason we can't do that is because of the old system, then why do we even need the old branches as we won't be using that anymore?

Also this is another thing that really should have been discussed with us before making a decision yourselves 😞

Reply:
winner => feature/6/<JIRA>